opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1006,6 +1008,55 @@ def set_up_accelerate_config(
                 "Failed to open config file for ml common upload: " + file_path + "\n"
             )
 
+    def _get_model_description_from_md_file(self, readme_file_path) -> str:
+        """
+        Get description of the model from README.md file
```

Comment:
Is the `README.md file` from huggingface? Can you add some example link?

Reply:
Yes, it's from huggingface. I can add example.